### DISCUSSION TITLE
Consigllio per gli acquisti per rendere la casa smart

### DISCUSSION THREAD

**Discussion by Asky82** [indomus/forum#3150] [Hardware & Firmware]
Ciao a tutti,

come da titolo, sto facendo un pò di ricerche per capire cosa acquistare per rendere la mia casa smart.

Nel dettaglio vorrei: 

Controllare luci interne ed esterne alla casa
Controllare il riscaldamento mediante valvole termostatiche e termostati
Avere sensori per porte/finestre
Sensori per le perdite d'acqua 
Sensori qualità aria

Qui chiaramente ci sono diverse possibilità con SONOFF,Shelly, BOSCH etc..

La sfida è che non so bene a quale standard affidarmi  per evitare di avere diversi HUB/reti forniti dalle diverse marche.
E possibilmente vorrei poter controllare tutto anche in locale sensa dover avere una connessione ad Internet.

Posso chiedervi un consiglio su cosa adopereste in questo caso?

Grazie 1000

**Comment by indomus** [comment]
Stai per commettere il più classico degli errori, ovvero quello che abbiamo fatto quasi tutti: partire dagli end device.
Errore. Devi partire PRIMA dall'HUB - e poi scegliere gli end device sarà molto più semplice, perché verrà fatto in funzione dei requisiti funzionali che avrai E dell'HUB stesso.
Parti da qui, ti serve mezz'ora: https://indomus.it/nozioni-di-base/

**Comment by Asky82** [comment]
Hai ragione ed ho già una risposta :)

Utilizzerò Home Assistant OS. Negli ultimi due mesi ci ho giocato parecchio
anche in versione core.
